### ISSUE TITLE
working with digest

### ISSUE THREAD

**Issue by rpajik** [aerospike/aerospike-client-php#46]
Hello,

maybe I do something wrong but. I get array of existing digest by this:

```
$status = $aerospike->scan('my_namespace','test_set', function ($record) use (&$digests) {
    $digests[] = $record["key"]["digest"];
});
```

When I want delete all of records from the set by this:

```
foreach ($digests as $digest) {
	$key = $aerospike->initKey('my_namespace','test_set',$digest,true);
	$aerospike->remove($key);
}
```

Some of records are not delete, because the digest which is included in $key is not complete, because some special characters ?
I have some examples ( firsts is var_dump of $digest and the 2nd is var_dump of $key ):

```
string(20) "üˆ¿G<†eèŽÔé=f" 
array(3) { ["ns"]=> string(12) "my_namespace" ["set"]=> string(6) "test_set" ["digest"]=> string(13) "üˆ¿G<†eè" } 

string(20) "ëÆ—¼ÀÓ„W²&¯Å€˜¢§ÃF" 
array(3) { ["ns"]=> string(12) "my_namespace" ["set"]=> string(6) "test_set" ["digest"]=> string(14) "ëÆ—¼ÀÓ„W²&¯Å€˜" } 

string(20) "í …RŠœØ©vá´ÅU²ž:" 
array(3) { ["ns"]=> string(12) "my_namespace" ["set"]=> string(6) "test_set" ["digest"]=> string(5) "í …RŠ" } 

string(20) "UnQ|uÐiŽðÖ=n»[BêÙ¡¢" 
array(3) { ["ns"]=> string(12) "my_namespace" ["set"]=> string(6) "test_set" ["digest"]=> string(4) "UnQ|" } 
```
I try some functions like addslahes() and so on, but it is not working. 

Interesting is, that if I bypass initKey function and generate $key array by myself, the digest string is the same as I receive by scan function byt the remove function still return error AEROSPIKE_ERR_RECORD_NOT_FOUND.

**Comment by rbotzer** [comment]
Thanks for reporting the bug. I can reproduce the same behavior.

The digest is a binary string so you shouldn't manipulate it with any of the string functions, but it should work as-is. We'll work to fix the problem.

**Comment by rbotzer** [comment]
This should be fixed with [release 3.3.16](https://github.com/aerospike/aerospike-client-php/releases/tag/3.3.16). Please verify.
